source/lib/omnitrace/library/config.cpp

```diff
@@ -1485,6 +1491,18 @@ get_use_roctracer()
 #endif
 }
 
+bool
+get_perfetto_roctracer_per_stream()
+{
+#if defined(OMNITRACE_USE_ROCTRACER) && OMNITRACE_USE_ROCTRACER > 0 &&                   \
+    defined(OMNITRACE_USE_PERFETTO) && OMNITRACE_USE_PERFETTO > 0
```

Comment:
```suggestion
#if defined(OMNITRACE_USE_ROCTRACER) && OMNITRACE_USE_ROCTRACER > 0
```

We always compile with perfetto (there is not even a cmake option to disable it) and that's extremely unlikely to change, even if we add support for different trace formats.